app/components/theme/switcher_component.yml

```
@@ -1,5 +1,5 @@
 link:
-  base: 'text-gray-700 group flex items-center text-sm dark:text-gray-300'
+  base: 'text-gray-700 group flex items-center dark:text-gray-300'
```

Just while we're here, I noticed the theme switch link text is a little darker than the other nav items. Would you mind updating that too please @sparshalc?

```suggestion
  base: 'text-gray-600 group flex items-center dark:text-gray-300'
```